.github/FUNDING.yml

```
@@ -0,0 +1,53 @@
+
+# Litecoin
+litecoin: ltc1qxy2kgdygjrsqtzq2n0yrf2493p83kkfjhx0wlh
+
+# Dogecoin
+dogecoin: D8j8v1GiRq3E9z4uP8Lm6sKcJbA2xY7wQ
+
+# Monero
+monero: 44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXg3ooDyLacg9ddETDm15fzEkUNkQz4o4Ek5GxTyuYt2bBGjeAPnc4
+
+# Zcash
+zcash: zs1ghe4yy9j0m6xncg33zgtn6nscqcnvfe9hw6vydv8qf7yznmusd70g3xw239dlqxtzjccf9crm6
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)